test/e2e/testsuite/testsuite.go

```diff
@@ -539,6 +549,54 @@ func FileExistsInCRCHome(fileName string) error {
 	return err
 }
 
+func BundleIsCachedOrNot(presetName string, isOrNot string) error {
+
+	bundle := ""
+	if presetName == "podman" {
+		bundle = constants.GetDefaultBundle(preset.Podman)
+	} else {
+		bundle = constants.GetDefaultBundle(preset.OpenShift)
+	}
+
+	theFile := filepath.Join(CRCHome, "cache", bundle)
```

Comment:
This might deserve a `getBundlePath(presetName)` helper.

Reply:
I put that function in the `util` package in `extended` folder. I then needed to redefine `CRCHome` variable whose scope didn't span to that package. Hopefully that's OK.